isb_web/manage.py

```diff
@@ -216,7 +220,13 @@ async def auth(request: starlette.requests.Request):
     token = await oauth.orcid.authorize_access_token(request)
     request.session["user"] = dict(token)
     redirect_url = url_utilities.joined_url(str(request.url), config.Settings().auth_response_redirect_fragment)
-    return starlette.responses.RedirectResponse(url=redirect_url)
+    # redirect_url = "http://localhost:3000/#/dois"
+    if "annotation" in request.query_params and request.query_params["annotation"]=="true":
+        # if login is for annotation purpose, add access token as query param 
+        redirect_url += "?access_token=" + token["access_token"]
```

Comment:
Going back to the webui(e.g. localhost:3000/#/) page instead of the dois page won't allow us to add the access token as a query param, as it clears all the query params and appends the previous search params. For now, even if we login for annotation purposes redirect to dois page.

Reply:
Do we need to make a fix to the `webui` side?

Reply:
It seems like we might want a generic landing page -- as we add more services we'll definitely want to route to more destinations after logging in.

Reply:
I agree. I added some fixes at `throughput-widget` to redirect to the landing page after saving the throughput tokens in the `dois` page.